cmd/completion_test.go

```diff
@@ -8,16 +8,14 @@ import (
 
 func TestCompletion(t *testing.T) {
 	t.Run("without correct args", func(t *testing.T) {
-		out, err := executeCommand(NewRootCmd(), "completion", "not-correct-arg")
+		_, err := executeCommand(NewRootCmd(), "completion", "not-correct-arg")
```

Comment:
Why this change? outstd will not show the error?

Reply:
This change is due to the enabling of the SilenceErrors property in the root command (https://github.com/mia-platform/miactl/blob/feature/select-custom-certificate/cmd/root.go#L32-L35).  
As a result, the error is not printed to std out automatically by cobra when the `executeCommand` function is run on selected command, requiring the test to be changed to just verify error content.

Nonetheless, due to these lines https://github.com/mia-platform/miactl/blob/feature/select-custom-certificate/cmd/root.go#L53-L56, the behaviour is retained.